cmd/bondsd/main.go

```diff
@@ -95,14 +110,13 @@ func exportAppStateAndTMValidators(
 ) (json.RawMessage, []tmtypes.GenesisValidator, error) {
 
 	if height != -1 {
-		app := simapp.NewSimApp(logger, db, traceStore, false, uint(1))
-		err := app.LoadHeight(height)
+		gApp := app.NewBondsApp(logger, db, traceStore, false, map[int64]bool{}, invCheckPeriod)
```

Comment:
Change `gApp` to `app`. I believe `gApp` refers to Gaia and that's not what we want.